media-sound/denemo/denemo-2.3.0-r1.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools xdg
+
+DESCRIPTION="A music notation editor"
+HOMEPAGE="http://www.denemo.org/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3+ OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# configure options currently not used:
+# --enable-mem(no) memory debugging: needs Electric fence (efence), which
+#		is not available in portage. See https://github.com/boundarydevices/efence
+# --enable-gtk-doc-pdf(no) doesn't work
+IUSE="alsa +aubio debug +evince jack +fluidsynth gtk-doc +gtk3 nls +portaudio
+	+portmidi +rubberband static test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/libxml2:2
+	>=dev-scheme/guile-2:12=
+	gnome-base/librsvg:2
+	media-libs/fontconfig:1.0
+	>=media-libs/libsmf-1.3
+	>=media-libs/libsndfile-1.0.28-r1
+	>=media-sound/lilypond-2.19.54[guile2(+)]
+	alsa? ( >=media-libs/alsa-lib-1.1.2 )
+	aubio? ( >=media-libs/aubio-0.4.1-r1:= )
+	evince? ( >=app-text/evince-3.22.1-r1:= )
+	fluidsynth? ( >=media-sound/fluidsynth-1.1.6-r1:= )
+	gtk3? (
+		x11-libs/gtk+:3
+		x11-libs/gtksourceview:3.0=
+	)
+	!gtk3? (
+		x11-libs/gtk+:2
+		x11-libs/gtksourceview:2.0
+	)
```

Comment:
Would it be a good opportunity to get rid of GTK2 support here?

Reply:
Good point. It should work, I'm going to check this.

Reply:
Had to remove gtk3 and evince options to work it out.